core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -1379,7 +1379,15 @@ public Cursor getFeedStatisticsCursor() {
     }
 
     /**
-     * Called when a database corruption happens
+     * Insert raw data to the database.     *
+     * Call method only for unit tests.
+     */
+    public void insertTestData(@NonNull String table, @NonNull ContentValues values) {
```

Comment:
How about adding the `@VisibleForTesting` annotation?

Reply:
I forgot about that, sorry.